src/hardware/input/mouse_config.cpp

```diff
@@ -270,6 +273,14 @@ static void config_init(Section_prop &secprop)
 	prop_bool->Set_help("If enabled, middle-click will release the captured mouse, and also capture\n"
 	                    "when seamless (enabled by default).");
 
+	prop_bool = secprop.Add_bool("mouse_multi_display_aware", always, true);
+	prop_bool->Set_help("Allows mouse seamless behavior and mouse pointer release to work in fullscreen\n"
+	                    "mode for systems with more than one display. (enabled by default).\n"
+	                    "Notes: In some cases (i.e. mirrored display or AV receiver connected via HDMI,\n"
+	                    "       but used for audio output only) DOSBox might be notified about multiple\n"
+	                    "       displays, while in reality only one screen output is available.\n"
+	                    "       In such case, disable the multi-display awareness.");
```

Comment:
Slightly reworded with more context up-front:
```
Note: You should set this to false if it incorrectly detects multiple 
displays when there's only one actually connected. This might happen 
if you're using mirrored display mode or using an AV receiver's HDMI 
input for audio-only listening.
```

Reply:
OK - but I have adapted your proposal for consistency with other help strings and changed the text a little - as with mirroring one actually has two displays (like laptop internal one + the projector), they just display same picture all the time.

Reply:
> OK - but I have adapted your proposal for consistency with other help strings

Even better!